dash/dash.py

```diff
@@ -1312,6 +1312,12 @@ def dispatch(self):
             inputs_state = inputs + state
             inputs_state = convert_to_AttributeDict(inputs_state)
 
+            if cb.get("no_output"):
+                outputs_list = []
+            elif not outputs_list:
+                # FIXME Old renderer support?
```

Comment:
Why do we want to support old renderer? Embedded? We should rebuild & test embedded with this update anyway.

Reply:
I'll look into it when I'll update embedded next.